android/src/main/kotlin/live/hms/hmssdk_flutter/HMSTrackExtension.kt

```diff
@@ -5,24 +5,31 @@ import live.hms.video.sdk.models.enums.HMSTrackUpdate
 
 class HMSTrackExtension {
     companion object{
-        fun toDictionary(track:HMSTrack?):HashMap<String,Any>?{
-            val hashMap=HashMap<String,Any>()
+        fun toDictionary(track:HMSTrack?):HashMap<String,Any?>?{
+            val hashMap=HashMap<String,Any?>()
             if(track==null)return null
+            hashMap["volume"]=null
```

Comment:
new params should always be added, below the current ones.
`hashMap["volume"]=null` should be added after Line 16